File: WebApplication1/WebApplication1/DTO/Request/ConcertReqDto.cs
using System.Text.Json.Serialization;
using WebApplication1.DTO.Response;
using WebApplication1.Models;
namespace WebApplication1.DTO.Request

{
    public class ConcertReqDto
    {
        public required string ConcertName { get; set; }
        public ConcertSpecReqDto? ConcertSpec { get; set; } = null;
        public string? CustomHall { get; set; }
        public HallBookingReqDto? HallBookingReq { get; set; } = null;
    }
}

File: angular-app/src/app/api/models/concert-req-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ConcertSpecReqDto } from './concert-spec-req-dto';
import { HallBookingReqDto } from './hall-booking-req-dto';
export interface ConcertReqDto {
  concertName: null | string;
  concertSpec?: ConcertSpecReqDto;
  customHall?: null | string;
  hallBookingReq?: HallBookingReqDto;
}
